po/eo.po

```diff
@@ -0,0 +1,236 @@
+# Esperanto translation for Ladybird.
+msgid ""
+msgstr ""
+"Project-Id-Version: ladybird wip\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-08-27 00:38+0300\n"
+"Language: eo\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 "
+"&& n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+
+#: window.ui:67
+msgid "...will appear here, once we implement them :^)"
+msgstr "...aperos ĉi tie, post kiam ni realigos ilin :^)"
+
+#: window.ui:166 window.ui:296
+msgid "Bookmarks"
+msgstr "Legosignoj"
+
+#: window.ui:66
+msgid "Bookmarks & History"
+msgstr "Legosignoj kaj Historio"
+
+#: window.ui:55
+msgid "Close _Other Tabs"
+msgstr "Fermi _Aliajn Langetojn"
```

Comment:
I don't think we're supposed to use title case (capitalizing nearly every word) everywhere in Esperanto, or in any languages other than English. Do other GNOME apps use title case?

Reply:
You're correct, and I did not do my research. I'll fix this now.